setup.py

```diff
@@ -189,7 +189,7 @@ def get_git_sha() -> str:
         "playwright": ["playwright>=1.37.0, <2"],
         "postgres": ["psycopg2-binary==2.9.6"],
         "presto": ["pyhive[presto]>=0.6.5"],
-        "trino": ["trino>=0.324.0"],
+        "trino": ["trino>=0.328.0"],
```

Comment:
@rusackas it seems like `trino` is included in `testing.in` and thus you'll have to recompile the requirements.